doc/integrations/services-routes-to-syndesis-connection.md

```diff
@@ -0,0 +1,86 @@
+# Integration Design
+
+## Auto discovery of services and routes that can become connections in Syndesis
+
+## Services / Resources Involved
+
+Syndesis
+
+OpenShift Routes / K8s services
+
+## Use Case
+
+As a developer working with OpenShift on applications I intend to be used by Syndesis, I would like a k8s native way to indicate that a service or route I have
+created should be available as a http(s) connection in Syndesis. I should also be able to indicate that I have a swagger definition
+and its location, in which case I would expect and API connection in Syndesis. I should not need to deploy this application to the same
+namespace as my Syndesis instance.
+
+# Proposed Approach
+
+## Resource Annotations
+There may be some existing annotations we could use below is an example of what these annotations would look like:
+```
+    integration/fuse-connection: "enabled"
+    integration/fuse-connection-swagger: "someswagerlocation"
```

Comment:
A port & protocol may also be needed.
A service may have multiple ports, and my be http or https